src/main/scala/cromwell/Main.scala

```diff
@@ -66,6 +67,10 @@ object Main extends App {
     argCapitalized = arg.capitalize
     action <- Actions.values find (_.toString == argCapitalized)
   } yield action
+
+  def deprecationNotice(): Unit = {
+    Console.err.println("This functionality is deprecated. Please use wdltool: https://github.com/broadinstitute/wdltool")
```

Comment:
Eventually the right thing to do is for both to state whichx version of the spec they support